snorkel/analysis/error_analysis.py

```diff
@@ -108,7 +109,7 @@ def __init__(self, null_pred: bool = False, null_gold: bool = False) -> None:
         null_gold
             If True, include the col corresponding to null gold labels
         """
-        self.counter = Counter()
+        self.counter: typing.Counter = Counter()
```

Comment:
I _think_ you can just do `from typing import Counter` and remove the `collections` import

Reply:
You thought correctly! Nice.